File: Server/JobModule/Models/JobInfoDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace JobModule.Models
{
    public class JobInfoDto
    {
        public DateTime AcceptedAt { get; set; }
        public DateTime FinishedAt { get; set; }
        public float Distance { get; set; }
        public bool IsLate { get; set; }
        public ulong PlannedIncome { get; set; }
        public long Income { get; set; }
        public Point Source { get; set; }
        public Point Destination { get; set; }
        public Cargo Cargo { get; set; }
        public bool IsDeliveried { get; set; }
        public long Penalty { get; set; }
    }
}

File: Client/app/src/main/java/com/nks/interactive/multimediapanel/models/job/JobInfoDto.kt
package com.nks.interactive.multimediapanel.models.job

import com.nks.interactive.multimediapanel.models.job.Point
import java.time.LocalDateTime

data class JobInfoDto(
    val acceptedAt: LocalDateTime,
    val finishedAt: LocalDateTime? = null,
    val distance: Float,
    val isLate: Boolean,
    val plannedIncome: Long,
    val income: Long,
    val source: Point,
    val destination: Point,
    val cargo: Cargo,
    val isDeliveried: Boolean,
    val penalty: Long
)